### PR TITLE
schinagl.ln version 2.9.3.4

### PR DESCRIPTION
**Pull request by schinagl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89771)

### CODE REVIEW

**Comment by schinagl** [comment]
I assumed an answer like this. 

It is an .exe without installer. Just the .exe. Copy it to a directory and call it from command prompt. 

You don't accept that. Correct?

**Comment by mdanish-kh** [comment]
> I assumed an answer like this.
> 
> It is an .exe without installer. Just the .exe. Copy it to a directory and call it from command prompt.
> 
> You don't accept that. Correct?

WinGet does accept such packages. The Installer type needs to be `portable` and not `exe`.

File: manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: schinagl.ln
+PackageVersion: 2.9.3.4
+Installers:
+- Architecture: x64
+  InstallerType: exe
```
Comment (mdanish-kh) [inline manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.installer.yaml:8]:
```suggestion
  InstallerType: portable
```

File: manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: schinagl.ln
+PackageVersion: 2.9.3.4
```
Comment (mdanish-kh) [inline manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.installer.yaml:5]:
```suggestion
PackageVersion: 2.9.3.4
Commands:
- ln
```
To be able to call the executable through command-line by `ln`

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: schinagl.ln
```
Comment (mdanish-kh) [inline manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.installer.yaml:4]:
```suggestion
PackageIdentifier: Schinagl.ln
```
Can you please make the capitalization consistent with #89774? With this PR and #89774, you are creating two directories `manifests/s/schinagl` and `manifests/s/Schinagl`

File: manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: schinagl.ln
```
Comment (mdanish-kh) [inline manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Schinagl.ln
```
Can you please make the capitalization consistent with https://github.com/microsoft/winget-pkgs/pull/89774? With this PR and https://github.com/microsoft/winget-pkgs/pull/89774, you are creating two directories manifests/s/schinagl and manifests/s/Schinagl

File: manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: schinagl.ln
```
Comment (mdanish-kh) [inline manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.yaml:4]:
```suggestion
PackageIdentifier: Schinagl.ln
```
Can you please make the capitalization consistent with https://github.com/microsoft/winget-pkgs/pull/89774? With this PR and https://github.com/microsoft/winget-pkgs/pull/89774, you are creating two directories `manifests/s/schinagl` and `manifests/s/Schinagl`

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
You would also need to change the directory name `manifests/s/schinagl` to `manifests/s/Schinagl` (notice the capitalization). Consequently the file names would also require re-naming to match the new PackageIdentifier. You could do this in a number of ways using git / GitHub.

A quick and easy way to do it would be to do it here inside GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open this PR in VSCode Online. Navigate to explorer menu, find the required files and update the directory and file structure. The directory should look similar to this:


After that, stage and commit the changes from the Source Control tab (the changes are automatically pushed)

File: manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.ln
+PackageVersion: 2.9.3.4
+PackageLocale: en-US
+Publisher: schinagl.priv.at
+PackageName: ln advanced command line tool
+License: MIT License
+Copyright: Copyright © 1999-2021 Hermann Schinagl
+ShortDescription: 'is a swiss army knife for hardlinks, symbolic links, junctions when it comes to copying. Further more it provides a incremental copy mechanism called Delorean Copy '
```
Comment (mdanish-kh) [inline manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml:11]:
```suggestion
ShortDescription: ln is a swiss army knife for hardlinks, symbolic links, junctions when it comes to copying. Further more it provides a incremental copy mechanism called Delorean Copy
```
I believe you wanted to write ln at the start? Also no need for single-quotes.

File: manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.ln
+PackageVersion: 2.9.3.4
+PackageLocale: en-US
+Publisher: schinagl.priv.at
```
Comment (mdanish-kh) [inline manifests/s/schinagl/ln/2.9.3.4/schinagl.ln.locale.en-US.yaml:7]:
```suggestion
Publisher: Schinagl
```
Just wanted to confirm whether this is intentional as you wrote `Schinagl` in the other PR https://github.com/microsoft/winget-pkgs/blob/591a98ea2f24bc0f0c4c8958a008e5e038cccfe5/manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml#L7

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
Please see https://github.com/microsoft/winget-pkgs/pull/89771#pullrequestreview-1194908027 so that the PR passes the validation pipelines.

**Comment by schinagl** [comment]
> You would also need to change the directory name `manifests/s/schinagl` to `manifests/s/Schinagl` (notice the capitalization). Consequently the file names would also require re-naming to match the new PackageIdentifier. You could do this in a number of ways using git / GitHub.
> 
> A quick and easy way to do it would be to do it here inside GitHub Web by pressing the . (dot) key on your keyboard that'll open this PR in VSCode Online. Navigate to explorer menu, find the required files and update the directory and file structure. The directory should look similar to this: 
> 
> After that, stage and commit the changes from the Source Control tab (the changes are automatically pushed)



> You would also need to change the directory name `manifests/s/schinagl` to `manifests/s/Schinagl` (notice the capitalization). Consequently the file names would also require re-naming to match the new PackageIdentifier. You could do this in a number of ways using git / GitHub.
> 
> A quick and easy way to do it would be to do it here inside GitHub Web by pressing the . (dot) key on your keyboard that'll open this PR in VSCode Online. Navigate to explorer menu, find the required files and update the directory and file structure. The directory should look similar to this: 
> 
> After that, stage and commit the changes from the Source Control tab (the changes are automatically pushed)

Still strugglin on how to get captial S in VS-Code. But will somehow make it

**Comment by mdanish-kh** [comment]
> Still struggling on how to get captial S in VS-Code. But will somehow make it

I was creating a video for it, but you managed to figure it out yourself 😅

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**